extensions/amazon-services/common/runtime/src/main/java/io/quarkus/amazon/common/runtime/FileTlsTrustManagersProvider.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.amazon.common.runtime;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import software.amazon.awssdk.http.TlsTrustManagersProvider;
+
+public class FileTlsTrustManagersProvider implements TlsTrustManagersProvider {
+
+    private final Path path;
+    private final String type;
+    private final char[] password;
+
+    public FileTlsTrustManagersProvider(FileStoreTlsManagersProviderConfig fileStore) {
+        path = fileStore.path.get();
+        type = fileStore.type.get();
+        password = fileStore.password.map(String::toCharArray).orElse(null);
+    }
+
+    @Override
+    public TrustManager[] trustManagers() {
+        try (InputStream storeInputStream = Files.newInputStream(path)) {
+            KeyStore keyStore = KeyStore.getInstance(type);
+            keyStore.load(storeInputStream, password);
+            TrustManagerFactory tmf = TrustManagerFactory.getInstance("SunX509");
```

Comment:
Maybe it's safer to use platform default trustmanager algorithm via `TrustManagerFactory.getDefaultAlgorithm()` ?